cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
\ No newline at end of file
+= Notable features in odo for the OCP 4.2 release
+
+{odo-title}{nbsp}{odo-ver} offers a simple way to develop applications on {product-title}. {odo-title} is completely client-based and requires no server within {product-title} cluster.
+
+{odo-title}{nbsp}{odo-ver} comes with:
+
+* Simple syntax and design centered around concepts familiar to developers such as projects, applications and components.
+* Compatibility with any language or runtime within the OpenShift catalog of component types.
+* Capability to add custom component types using custom image builders.
+* Official support for Java and Node.js component types.
+* Service catalog which allows users to connect applications deployed on {product-title} cluster to the Template Service Broker.
+* `odo service create` - an interactive mode which guides users through the process of creating applications.
+* `odo watch` - a feature that automatically detects changes to local code and applies the changes to the component in real time.
+
+////
+
+[id="odo-getting-support_{context}"]
+== Getting support
+
+.For Documentation
+
+If you have found an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+
+.For Product
+
+If you have found an error, encountered a bug or have suggestions for improving the functionality of {odo-title}, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *Red Hat odo for OpenShift Container Platform* product type.
+
+Provide as many details in the description of the issue as possible.
+
+[id="odo-fixed-issues_{context}"]
+== Fixed issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760573[Bug 1760573] The active pointer does not switch to the current active project after the deletion of a project.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760578[Bug 1760578] The `odo watch` command does not fail with an error message for git based components, but it should.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760583[Bug 1760583] The `odo config unset` command does not unset an environment variable, but says it did.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760585[Bug 1760585] The `odo delete --all` command deletes the `$HOME/.odo` folder when run from `$HOME`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760589[Bug 1760589] Autocompletion does not work for the `--context` flag in `odo push`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1761442[Bug 1761442] The `component create` command fails when used with `--context` flag and `--binary` flag if the binary is located in a temporary folder.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1783179[Bug 1783179] When you set an environment variable, create an URL route and then change the source code of the component, the URL route becomes inaccessible.
```

Comment:
s/create an/create a/
Also, this sentence is confusing. It sounds like "create a URL route" is an instruction. I think it's missing a comma, so try something like, "When you set an environment variable, create a URL route, and change the source code of the component, the URL route becomes inaccessible."